src/main/java/com/rogerr/custom/model/User.java

```diff
@@ -0,0 +1,83 @@
+package com.rogerr.custom.model;
+
+public class User {
+
+	private Long id;
+
+	private String username;
+
+	private String address;
+
+	private String email;
+
+	public User() {
+	}
+
+	public User(Long id, String username, String address, String email) {
+		this.id = id;
+		this.username = username;
+		this.address = address;
+		this.email = email;
+	}
+
+	public Long getId() {
+		return id;
+	}
+
+	public void setId(Long id) {
+		this.id = id;
+	}
+
+	public String getUsername() {
+		return username;
+	}
+
+	public void setUsername(String username) {
+		this.username = username;
+	}
+
+	public String getAddress() {
+		return address;
+	}
+
+	public void setAddress(String address) {
+		this.address = address;
+	}
+
+	public String getEmail() {
+		return email;
+	}
+
+	public void setEmail(String email) {
+		this.email = email;
+	}
+
+	public int hashCode() {
+		final int prime = 31;
+		int result = 1;
+		result = prime * result + (int) (id ^ (id >>> 32));
+		return result;
+	}
+
+	public boolean equals(Object obj) {
+		if (this == obj)
+			return true;
+		if (obj == null)
+			return false;
+		if (!(obj instanceof User))
+			return false;
+		User other = (User) obj;
+		if (id != other.id)
+			return false;
+		return true;
+	}
```

Comment:
don't overload the hashCode or equals() methods unless you have a specific reason to do so.